browser/css/jsdialogs.css

```diff
@@ -2400,10 +2400,25 @@ kbd,
 }
 
 /* Third column has nothing to display */
-#HyperlinkMailPage #grid1 {
-	grid-template-columns: repeat(2, auto) 0px !important;
+#\~Mail #grid1 {
+	grid-template-columns: 1fr 3fr 0px !important;
+	column-gap: 0px !important;
 }
 
+#\~Internet #grid1, #\~Internet #grid3, #\~Mail #grid3 {
+	grid-template-columns: 1fr 3fr !important;
+	column-gap: 0px !important;
+}
+
```

Comment:
I'd suggest we optimize the CSS selectors for better performance. Using `#HyperlinkInternetPage #grid1, #HyperlinkInternetPage #grid3, #HyperlinkMailPage #grid3` would be more efficient since simple ID selectors process faster than escaped characters.

cc @eszkadev